webui/react/src/pages/Dashboard.tsx

```diff
@@ -57,6 +61,22 @@ const Dashboard: React.FC = () => {
   });
   const [ filters, setFilters ] = useState<TaskFilters>(initFilters);
 
+  const fetchExperiments = useCallback(async (): Promise<void> => {
+    try {
+      const response = await getExperimentList(
+        { descend: true, key: 'startTime' },
+        { limit: 100, offset: 0 },
+        { showArchived: false, states: filters.states, username: filters.username },
+      );
+      const experiments = decodeExperimentList(response.experiments || []);
+      setExperiments(experiments);
+    } catch (e) {
+      handleError({ message: 'Unable to fetch experiments.', silent: true, type: ErrorType.Api });
```

Comment:
question: I don't understand the semantics of `handleError`, but any reason we want to silently fail here? Is this for transient failures?

Reply:
there is already a handler that's at the `getExperimentList` level that is not silent. This error handler serves to capture any other issues but instead of broadcasting this one, this will just log it to segment.